packs/data/blood-lords-bestiary.db/nightmare-portal.json

```diff
@@ -46,7 +46,7 @@
     ],
     "name": "Nightmare Portal",
     "prototypeToken": {
-        "name": "RENAMEME"
+        "name": "Nightmare Portal"
```

Comment:
You can wipe the whole `prototypeToken` property in this case (which would happen on extract)

Reply:
Done